registry/api/v2/descriptors.go

```diff
@@ -374,7 +346,7 @@ var routeDescriptors = []RouteDescriptor{
 									ContentType: "application/json; charset=utf-8",
 									Format:      errorsBody,
 								},
-								ErrorCodes: []ErrorCode{
+								ErrorCodes: []errcode.ErrorCode{
```

Comment:
With the change to the StatusCode to ErrorCode mapping, we only need one or the other here. This will require some updates to the api template.